devchat.go

```diff
@@ -57,6 +60,20 @@ type room struct {
 	usersMutex sync.Mutex
 }
 
+// don't forget to update savePrefs() and loadPrefs() if you change this!
+type userexport struct {
```

Comment:
I would name this type something like `userSaveData`

Reply:
My logic being that it's the user struct with exported fields

Reply:
It doesn't convey intention, though. Also, could we solve the problem by instead exporting some fields in the regular `user` type and JSON-ing that so that exported fields are automatically part of autoload?

Reply:
Well, yes, but given that none were exported at all, to me it seemed smarter to make a new struct

Reply:
Okay, could you export the autoload-able fields inside `user`? That would make code less repetitive and easier to maintain. Also feels a bit cleaner, right?

Reply:
done.